src/components/toolbar/toolbar.js

```diff
@@ -122,7 +122,7 @@ function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
          *
          */
         function onChangeScrollShrink(shrinkWithScroll) {
-          var closestContent = element.parent().find('md-content');
+          var closestContent = angular.element($mdUtil.getSiblings(element, 'md-content'));
```

Comment:
It might be more convenient `getSiblings` returned a jqLite wrapper in the first place, e.g.

```js
var siblings Array.prototype.filter.call(element.parentNode.children, ...);
return angular.element(siblings);
```